### PR TITLE
[hi] Localize kubernetes.md to Hindi

### LINKED ISSUE #1588: [hi] Localize kubernetes.md to Hindi
content: [https://glossary.cncf.io/kubernetes/](https://glossary.cncf.io/kubernetes/) 

Please complete DCO requirements before making a PR or commits : [https://wiki.linuxfoundation.org/dco](https://wiki.linuxfoundation.org/dco)

### PR DESCRIPTION
**Pull request by Omkar0114**
Signed-off-by: Omkar kulkarni <88308267+Omkar0114@users.noreply.github.com>

Localize kubernetes.md to Hindi

resolve #1588

- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by Omkar0114** [comment]
Hey, @Garima-Negi , @jayesh-srivastava , @sayantani11 @anubha-v-ardhan can you take a look on this PR? And suggest any changes !

File: content/hi/kubernetes.md
Comment (anubha-v-ardhan) [inline content/hi/kubernetes.md:5]:
```suggestion
---
title: कुबेरनेट्स (Kubernetes)
status: Completed
category: प्रौद्योगिकी
tags: ["infrastructure", "fundamental", ""]
---
```

File: content/hi/kubernetes.md
Comment (anubha-v-ardhan) [inline content/hi/kubernetes.md:17]:
```suggestion
## समस्या 
```

File: content/hi/kubernetes.md
Comment (anubha-v-ardhan) [inline content/hi/kubernetes.md:22]:
```suggestion
## समाधान
```

File: content/hi/kubernetes.md
Comment (anubha-v-ardhan) [inline content/hi/kubernetes.md:10]:
```suggestion
यह एक "डेटासेंटर ऑपरेटिंग सिस्टम" के रूप में कार्य करते हुए आधुनिक बुनियादी ढांचे पर कंटेनरीकृत एप्लीकेशन के जीवनचक्र को स्वचालित करता है जो एक [वितरित प्रणाली](/distributed-systems/) में एप्लीकेशन का प्रबंधन करता है।
```

File: content/hi/kubernetes.md
Comment (anubha-v-ardhan) [inline content/hi/kubernetes.md:12]:
```suggestion
कुबेरनेट्स [कंटेनर](/container/) को [नोड्स](/nodes/) में एक [क्लस्टर](/cluster/) में शेड्युल करता है। कंटेनरीकृत एप्लीकेशन को चलाने के लिए लोड बैलेंसर, आग्रही (persistent) स्टोरेज, आदि जैसे कई बुनियादी ढांचे के संसाधनों को बंडल करता है।
```

File: content/hi/kubernetes.md
Comment (anubha-v-ardhan) [inline content/hi/kubernetes.md:14]:
```suggestion
कुबेरनेट्स स्वचालन और विस्तारणीयता को सक्षम करता है, जिससे उपयोगकर्ता पुनरुत्पादित तरीके से घोषणात्मक रूप से (नीचे देखें) एप्लीकेशन को तैनात कर सकते हैं।
```

File: content/hi/kubernetes.md
Comment (anubha-v-ardhan) [inline content/hi/kubernetes.md:31]:
```suggestion
कुबेरनेट्स [स्वयं](/self-healing/) ठीक हो जाता है।
```

File: content/hi/kubernetes.md
Comment (Omkar0114) [reply content/hi/kubernetes.md]: Updated :)

File: content/hi/kubernetes.md
Comment (Omkar0114) [reply content/hi/kubernetes.md]: updated

File: content/hi/kubernetes.md
Comment (Omkar0114) [reply content/hi/kubernetes.md]: Updated :+1:

File: content/hi/kubernetes.md
Comment (Omkar0114) [reply content/hi/kubernetes.md]: updated

**Comment by Omkar0114** [comment]
Hi, @anubha-v-ardhan Updated docs with your suggestions, have a look once. Suggest any changes if needed :)

**Review by anubha-v-ardhan — APPROVED** [review]
LGTM!

Please wait for one more approval from another maintainer before we can merge this.

**Comment by Omkar0114** [comment]
Hi, one approved is there! Anyone can review it @Garima-Negi @sayantani11 @jayesh-srivastava

**Comment by Omkar0114** [comment]
Hey @anubha-v-ardhan @sayantani11 @Garima-Negi @jayesh-srivastava Anyone can review this PR as one LGTM has been added!

File: content/hi/kubernetes.md
Comment (aj11anuj) [inline content/hi/kubernetes.md:11]:
```suggestion
यह एक "डेटासेंटर ऑपरेटिंग सिस्टम" के रूप में कार्य करते हुए आधुनिक इन्फ्रा़स्ट्रक्चर पर कंटेनरीकृत एप्लीकेशन के जीवनचक्र को स्वचालित करता है जो एक [वितरित प्रणाली](/distributed-systems/) में एप्लीकेशन का प्रबंधन करता है।
```

File: content/hi/kubernetes.md
Comment (aj11anuj) [inline content/hi/kubernetes.md:13]:
```suggestion
कुबेरनेट्स [कंटेनर](/container/) को [नोड्स](/nodes/) के एक [क्लस्टर](/cluster/) में शेड्युल करता है। कंटेनरीकृत एप्लीकेशन को चलाने के लिए लोड बैलेंसर, सतत (persistent) स्टोरेज, आदि जैसे कई इन्फ्रा़स्ट्रक्चर के संसाधनों को बंडल करता है।
```

File: content/hi/kubernetes.md
Comment (aj11anuj) [inline content/hi/kubernetes.md:21]:
```suggestion
जैसे-जैसे कम्प्यूट संसाधनों की मांग बढ़ती है और संगठनों को कम इंजीनियरों के साथ अधिक आयोजन करने की क्षमता प्रदान करने की आवश्यकता होती है, उस मांग को पूरा करने के लिए नई तकनीकों और कार्य विधियों की आवश्यकता होती है।
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
Some suggestions from my side

**Review by aj11anuj — APPROVED** [review]
Lgtm 👍

**Review by kumarankit999 — APPROVED** [review]
lgtm

**Pull request merged**
